tempesta_fw/http_sess_conf.c

```diff
@@ -211,13 +235,31 @@ tfw_cfgop_cookie_set(TfwCfgSpec *cs, TfwCfgEntry *ce)
 					 " attribute: '%s'\n", cs->name, val);
 				return -EINVAL;
 			}
+		} else if (!strcasecmp(key, "path")) {
+			if (tfw_cfgop_cookie_set_path(sticky, val))
+			{
+				T_ERR_NL("%s: invalid value for 'path'"
+					 " attribute: '%s'\n", cs->name, val);
+				return -EINVAL;
+			}
+			was_path = true;
 		} else {
 			T_ERR_NL("%s: unsupported attribute: '%s=%s'.\n",
 				 cs->name, key, val);
 			return -EINVAL;
 		}
 	}
 
+	if (!was_path) {
+		static const char *dflt_path = "path=/";
+		sticky->options_str[sticky->options.len + 0] = ';';
+		sticky->options_str[sticky->options.len + 1] = ' ';
+		memcpy(&sticky->options_str[sticky->options.len + 2], dflt_path,
```

Comment:
Why not combine dilimeters and the attribute name? `dflt_path = "; path=/";`. A single `memcpy` will handle that.